lib/events.js

```diff
@@ -202,8 +387,7 @@ EventEmitter.prototype.emit = function emit(type) {
   return true;
 };
 
-EventEmitter.prototype.addListener = function addListener(type, listener) {
-  var m;
+EventEmitter.prototype.on = function on(type, listener) {
```

Comment:
Another superfluous change not directly related to the PR. I'm find w/ it being here, but not in the same commit.